mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -888,6 +888,8 @@ node_pool_auto_config {
 
 The `node_config_defaults` block supports:
 
+* `logging_variant` (Optional) The type of logging agent that is used as the default value for node pools in the cluster. Valid values include DEFAULT and MAX_THROUGHPUT.
```

Comment:
You'll need to update this file to indicate that `node_pool_defaults` and `node_config_defaults` are both no longer beta, and that gcfs_config _is_ still beta-only